robots/simulatedAckermanCar.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
```

Comment:
This is a minor nit, but could you rename this to ``simulated_ackerman_car.h``? Just because that's the naming convention we've used everywhere else.